File: Source/Core/Utility/UIDevice+Utility.m
//
//  UIDevice+Utility.m
//  Eunomia
//
//  Created by Ian on 3/14/15.
//  Copyright (c) 2015 Eunomia. All rights reserved.
//

#import "UIDevice+Utility.h"

@implementation UIDevice (Eunomia_Utility)

- (BOOL)isSimulator
{
    return [self.model isEqualToString:@"iPhone Simulator"];
}

- (NSString *)vendorUUIDString
{
    return self.identifierForVendor.UUIDString;
}

@end

File: Source/Core/UtilityExtensions/UIDevice+Utility.swift
//
//  UIDevice+Utility.swift
//  Eunomia
//
//  Created by Ian Grossberg on 7/24/15.
//  Copyright (c) 2015 Adorkable. All rights reserved.
//

import UIKit

extension UIDevice
{
    public var isSimulator : Bool {
        #if targetEnvironment(simulator)
            return true
        #else
            return false
        #endif
    }
    
    public var vendorUUIDString : String? {
        return self.identifierForVendor?.uuidString
    }
}
